frontend/benefit/handler/.eslintrc.js

```diff
@@ -0,0 +1,47 @@
+const { join } = require('path');
+
+module.exports = {
```

Comment:
to avoid copy-paste, this is only javascript object which could be partially imported from for example `frotend/.eslinrc.js` root folder and then overwrite the parts which are project-specified.
Something like
```
const baseConfig = require ('../../eslintrc.js`

module.exports = {
    ...baseConfig,
    rules: {
       ...baseConfig.rules,
       (custom rules)
    },
    overrides: [ ...baseConfig.overrides, (custom overrides) ],
}
```

Reply:
good point

Reply:
but when comparing even kesaseteli one and applicant not that much in common, more customisations then "base". lets leave it for right now.